inference/core/roboflow_api.py

```diff
@@ -183,19 +181,176 @@ def get_roboflow_model_data(
     endpoint_type: ModelEndpointType,
     device_id: str,
 ) -> dict:
-    api_url = wrap_url(
-        f"{API_BASE_URL}/{endpoint_type.value}/{model_id}?api_key={api_key}&device={device_id}&nocache=true&dynamic=true"
+    api_url = _add_params_to_url(
+        url=f"{API_BASE_URL}/{endpoint_type.value}/{model_id}",
+        params=[
+            ("api_key", api_key),
+            ("nocache", "true"),
+            ("device", device_id),
+            ("dynamic", "true"),
+        ],
+    )
+    return _get_from_url(url=api_url)
+
+
+@wrap_roboflow_api_errors()
+def get_roboflow_active_learning_configuration(
```

Comment:
This is the place holder waiting for updates on the Roboflow main app back end?

Reply:
yes, exactly